__tests__/test.js

```diff
@@ -17,6 +18,82 @@ describe('filterResponseForImportantEvents', () => {
     });
 });
 
+describe('fetchPageOfDataAndFilter', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
```

Comment:
Seeing this code inspired me to create this issue: https://github.com/indeedeng/starfish/issues/78